code/go/internal/validator/semantic/types.go

```diff
@@ -92,13 +92,14 @@ func (r *runtimeField) UnmarshalYAML(value *yaml.Node) error {
 }
 
 type field struct {
-	Name       string `yaml:"name"`
-	Type       string `yaml:"type"`
-	Unit       string `yaml:"unit"`
-	DateFormat string `yaml:"date_format"`
-	MetricType string `yaml:"metric_type"`
-	Dimension  bool   `yaml:"dimension"`
-	External   string `yaml:"external"`
+	Name       string   `yaml:"name"`
+	Type       string   `yaml:"type"`
+	Unit       string   `yaml:"unit"`
+	DateFormat string   `yaml:"date_format"`
+	MetricType string   `yaml:"metric_type"`
+	Dimension  bool     `yaml:"dimension"`
+	Dimensions []string `yaml:"dimensions"`
```

Comment:
The resemblance of `dimension` and `dimensions` are very close. Could there be a better name so that developers attempt for an incorrect usage? Unfortunately, presently, i can't think of a better name either.